src/net/NetworkClient.h

```diff
@@ -5,64 +5,71 @@
 #include <boost/asio/ssl.hpp>
 #include "util/Datagram.h"
 
```

Comment:
@AlJaMa Presumably somebody hacking on this low-level of Astron would know that. If there is high-level documentation of the network architecture somewhere, this should go there; is a comment for this purpose ideal?